### PR TITLE
Setup react application

### PR DESCRIPTION
**Pull request by tnagorra**
- Add configurations for:
    - webpack
    - babel
    - postcss
    - eslint
    - stylelint
    - typescript

### CODE REVIEW

File: src/App/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Lyrics from '#components/Lyrics';
+
+import styles from './styles.css';
+
+function App() {
+    return (
+        <div className={styles.app}>
+            <div>
```
Comment (barshathakuri) [inline src/App/index.tsx:10]: why there is a blank div?

File: src/App/styles.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    display: flex;
+    position: relative;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    z-index: auto;
+    background-color: #ddffff;
+    width: 100vw;
+    height: 100vh;
+    color: 'dddddd';
```
Comment (barshathakuri) [inline src/App/styles.css]: Please add # in the color code.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            if (self === top) {
+                var antiClickjack = document.getElementById("anti-click-jacking");
+                antiClickjack.parentNode.removeChild(antiClickjack);
+            } else {
+                top.location = self.location;
+            }
+        </script>
+        <!-- End of anti-click-jacking -->
+
+        <!-- Start of service-worker -->
```
Comment (barshathakuri) [inline src/index.html:41]: Remove these comments

**Review by priyesh777 — APPROVED** [review]

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+                {
+                    test: /\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/i,
+                    exclude: /(node_modules)/,
+                    include: getPath('src/'),
+                    type: 'asset/resource',
+                    /*
+                    options: {
+                        name: isProduction
+                            ? 'assets/[name].[contenthash]][ext]',
+                            : 'assets/[name].[ext]',
```
Comment (barshathakuri) [inline webpack.config.js:108]: Please Remove unwanted comments

File: src/App/index.tsx (hunk 1 above)
Comment (tnagorra) [reply src/App/index.tsx:10]: It is not a blank div.

File: src/App/styles.css (hunk 2 above)
Comment (tnagorra) [reply src/App/styles.css]: Nice catch!

File: src/index.html (hunk 3 above)
Comment (tnagorra) [reply src/index.html:41]: These are proper comments.
When we request to remove comments, it's generally commented out code.

File: webpack.config.js (hunk 4 above)
Comment (tnagorra) [reply webpack.config.js:108]: The behavior of asset/resource is still not known, that's what commenting out the default configuration.
Will remove after assets are added and the behavior is confirmed.

Comment (barshathakuri) [reply webpack.config.js:108]: okay

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**